addons/TBMod_bw/config.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+    Part of the TBMod ( https://github.com/TacticalBaconDevs/TBMod )
+    Developed by http://tacticalbacon.de
+*/
+class CfgPatches
+{
+    class TBMod_bw
+    {
+        name = "TBMod BW";
+
+        requiredAddons[] = {
+            "TBMod_main",
+            "bwa3_g82",
+            "ace_advanced_ballistics",
```

Comment:
wird schon im TBMod_main required, denke aber das macht jetzt keine Probleme...

Reply:
in Ordnung. heißt alles, was im TBMod_main vorausgesetzt wird; muss in einem Unterordner wie TBMod_bw nicht mehr reingeschrieben werden